unocss.config.ts

```diff
@@ -11,7 +11,7 @@ import {
 
 export default defineConfig({
   shortcuts: [
-    ['btn', 'px-4 py-1 rounded inline-block bg-teal-600 text-white cursor-pointer hover:bg-teal-700 disabled:cursor-default disabled:bg-gray-600 disabled:opacity-50'],
```

Comment:
Have actually no issue in dark mode with this configuration. Is it ok for you? Otherwise I delete these changes.
<img width="1792" alt="Screenshot" src="https://user-images.githubusercontent.com/50230578/168829858-e3b3bed5-2008-4e11-90ac-9e431e9a9046.png">